app/Handlers/Reference/LaravelHandler.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Handlers\Reference;
+
+use App\Slack\Event;
+use App\Slack\Message;
+use App\Handlers\Handler;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+
+final class LaravelHandler extends Handler
+{
+
+    private $client;
+
+    /**
+     * Loads GuzzleHttp Client.
+     * @param Client $client
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canHandle(Event $event)
+    {
+        return
+            $event->isMessage() &&
+            ($event->isDirectMessage() || $event->mentions($this->eve->userId())) &&
+            $event->matches('/\b(laravel)\b/i')
+            ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(Event $event)
+    {
+        $versions = [
+            'master', '5.3', '5.2', '5.1', '5.0', '4.2'
+        ];
+
+        $words = explode(' ',$event->text());
+
+        if(in_array($words[2], $versions)) {
+            $version = $words[2];
+            $query = $words[3];
+        } else {
+            $version = $versions[1];
+            $query = $words[2];
+        }
+
+        try {
+            $url = "https://laravel.com/docs/$version/$query";
+            $this->client->get($url);
```

Comment:
Can you do a different type of request here (maybe a `HEAD` request) so that we don't need to download the entire page to check it exists?